tests/test-war-smoke/pom.xml

```diff
@@ -152,18 +93,22 @@
     </profile>
     <profile>
       <id>test.remote</id>
-      <activation>
-        <property>
-          <name>test.mode</name>
-          <value>remote</value>
-        </property>
-      </activation>
+      <properties>
+        <!-- change the build timestamp to something palatable for gcloud -->
+        <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>

```

Comment:
We need to change it there as well because underscores are no longer allowed in the version tags.